hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -453,14 +453,14 @@ message ContainerBalancerConfigurationProto {
 }
 
 message DiskBalancerConfigurationProto {
-    optional double threshold = 1;
-    optional double diskBandwidth = 2;
+    optional string threshold = 1;
+    optional uint64 diskBandwidth = 2;
```

Comment:
@ferhui Updated the type here, please have a look.

Reply:
Got it, thank you!